src/main/java/uk/gov/cshr/controller/signup/SignupController.java

```diff
@@ -93,14 +99,24 @@ public String sendInvite(Model model,
         }
 
         final String email = form.getEmail();
-
-        if (inviteRepository.existsByForEmailAndStatus(email, InviteStatus.PENDING)) {
+        Optional<Invite> invite = inviteService.findByForEmailAndStatus(email, InviteStatus.PENDING);
+        if(invite.isPresent()) {
+            if (!inviteService.ifExpired(invite.get())) {
+                long timeForReReg = new Date().getTime() - invite.get().getInvitedAt().getTime();
+                if (timeForReReg < allowedTimeForReReg * 1000) {
+                    log.info("{} user given time to wait before re-registration", email);
```

Comment:
Can we change this info log message to be more informative e.g. "<User> can't  re-register within <configured> time."

Reply:
Done